live/root/usr/lib/dracut/modules.d/99agama-cmdline/agama-cmdline-conf.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/sh
+
+[ -e /dracut-state.sh ] && . /dracut-state.sh
+
+. /lib/dracut-lib.sh
+
+get_agama_args() {
+	local _i _found
+
+	for _i in $CMDLINE; do
+		case $_i in
+		LIBSTORAGE_* | YAST_* | agama* | Y2* | ZYPP_*)
+			_found=1
+			;;
+		esac
+
+		if [ -n "$_found" ]; then
+			printf "Agama variable found ($_i)"
+			printf $_i
```

Comment:
NP: wasn't `$_i` already printed in the line before?

Reply:
```suggestion
```